<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextEditor::TabPreferencesWidget</class>
 <widget class="QWidget" name="TextEditor::TabPreferencesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>122</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TextEditor::FallbackSelectorWidget" name="fallbackWidget" native="true"/>
   </item>
   <item>
    <widget class="TextEditor::TabSettingsWidget" name="tabSettingsWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditor::TabSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">texteditor/tabsettingswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextEditor::FallbackSelectorWidget</class>
   <extends>QWidget</extends>
   <header location="global">texteditor/fallbackselectorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
